trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table.njk

```diff
@@ -2,56 +2,95 @@
+        {% endif %}
+
+        <tr class="govuk-table__row" data-cy="utilisation-report-reconciliation-table-row-bank-{{ summaryItem.bank.id }}">
+          <th
+            scope="row"
+            class="govuk-table__header govuk-!-font-weight-regular"
+            data-sort-value="{{ summaryItem.bank.name }}"
+          >
+            <div class="govuk-checkboxes govuk-checkboxes--small">
+              <div class="govuk-checkboxes__item" data-cy="checkbox--{{ id }}">
+                <input class="govuk-checkboxes__input govuk-!-top-0" id="{{ checkboxId }}" name="{{ checkboxId }}" type="checkbox"/>
+                <label class="govuk-label govuk-checkboxes__label" for="">
+                  <a class="govuk-link govuk-link--no-visited-state" href="#">{{ summaryItem.bank.name }}</a>
+                </label>
+              </div>
+            </div>
+          </th>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.displayStatus }}">
+            {{ reconciliationStatus.render({
+              status: summaryItem.status,
+              displayStatus: summaryItem.displayStatus
+            }) }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.dateUploaded }}">
+            {{ summaryItem.formattedDateUploaded }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.totalFeesReported }}">
+            {{ summaryItem.totalFeesReported }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.reportedFeesLeftToReconcile }}">
+            {{ summaryItem.reportedFeesLeftToReconcile }}
+          <td class="govuk-table__cell">
+            {% if summaryItem.downloadPath %}
+              <a
+                class="govuk-link"
+                href="{{ summaryItem.downloadPath }}"
+                aria-label="Download {{ summaryItem.bank.name }} report"
+              >
+                Download
+              </a>
+            {% endif %}
+          </td>
+        </tr>
+      {% endfor %}
+      </tbody>
+    </table>
+
+    <p class="govuk-body">
+      When a report has been reconciled, select it and mark it as completed. If it needs to be returned to its previous state, mark it as not completed.
+    </p>
+
+    <div class="govuk-button-group">
+      <input
+        formaction="?_csrf={{ csrfToken }}&status=completed&month={{ month }}&year={{ year }}"
+        type="submit"
+        class="govuk-button govuk-!-margin-right-1"
+        data-module="govuk-button"
+        value="Mark report as completed"
+        data-cy="mark-report-as-completed-button"/>
+      <input
+        formaction="?_csrf={{ csrfToken }}&status=not-completed&month={{ month }}&year={{ year }}"
+        type="submit"
+        class="govuk-button govuk-button--secondary govuk-!-margin-right-1"
+        data-module="govuk-button"
+        value="Mark as not completed"
+        data-cy="mark-as-not-completed-button"/>
```

Comment:
I think we can simplify these a bit by using `govukButton` (see other uses of `formaction` in the codebase) which uses an underlying `button` element. I think `<input type="button"` or `<input type="submit"` was used historically due to compatibility issues with older browsers (like IE 6), but the default now should just be to use `button` (or in our case we can use the `govukButton` nunjucks template).

Reply:
I used this format as `utlisation-report-upload` in `portal` does this rather than using the `govukButton` component, but I will look into it and see if I can find any reason to not use it

Reply:
Looks like `govukButton` should work fine